packages/big-design/src/components/Button/styled.tsx

```diff
@@ -29,7 +29,6 @@ export const StyledButton = styled.button<ButtonProps & MarginProps>`
   line-height: ${({ theme }) => theme.lineHeight.xLarge};
   outline: none;
   padding: ${({ theme }) => `0 ${theme.spacing.medium}`};
-  pointer-events: ${({ isLoading }) => (isLoading ? 'none' : 'auto')};
```

Comment:
`isLoading` sets the button as `disable` now, no need to set `pointer-events` manually